ruleset/rulesetHelper.go

```diff
@@ -79,32 +83,25 @@ func GetRuleIdsToRun(configFile *config.Config, opts *options.Options) ([]rules.
 	}
 	// If user has specified specific rules, just return those
 	if opts.Rules != "" {
-		ruleIds := opts.SpecificRuleIds()
-		if err := validateRuleIds(ruleIds); err != nil {
-			return nil, nil, err
```

Comment:
What will happen now if user provides incorrect rule? 
Shouldn't it print a warning that rule Ids are not valid?